pkg/application/create.go

```diff
@@ -91,16 +105,26 @@ func handleSingleRepo(op CreateOp, githubClient *github.Client, localRepo *git.L
 	}, nil
 }
 
-func handleMonorepo(op CreateOp, githubClient *github.Client, localRepo *git.LocalRepository) (result CreateResult, err error) {
+func (svc *Service) handleMonorepo(op CreateOp, localRepo *git.LocalRepository) (result CreateResult, err error) {
 	branchName := "add-" + op.Name
 	if err := checkoutNewBranch(localRepo, branchName); err != nil {
 		return result, err
 	}
 
-	if err := renderTemplateMaybe(op); err != nil {
+	additionalArgs := []template.Argument{
+		{
+			Name:  "working_directory",
+			Value: "./" + op.Name,
+		},
+		{
+			Name:  "version_prefix",
```

Comment:
I think we should add those to ImplicitParameters, so they are also handled with `template render`.

I also not that sure about specifying implicit parameters in `template.yaml`. Or at least we should handle it somehow in case we get duplicated parameters (allow override it or something)

Reply:
added to implicit params